crates/parser/src/codegen.rs

```diff
@@ -87,4 +87,18 @@ mod tests {
             ],
         )
     }
+
+    #[test]
+    fn test_create_domain() {
+        test_get_node_properties(
+            "create domain us_postal_code as text check (value is not null);",
+            SyntaxKind::CreateDomainStmt,
+            vec![
+                TokenProperty::from(SyntaxKind::Create),
+                TokenProperty::from(SyntaxKind::DomainP),
+                TokenProperty::from(SyntaxKind::As),
+                TokenProperty::from("us_postal_code".to_string()),
```

Comment:
Here, for the range `0..32` I would expect this order:

```rust
TokenProperty::from(SyntaxKind::Create),
TokenProperty::from(SyntaxKind::DomainP),
TokenProperty::from("us_postal_code".to_string()),
TokenProperty::from(SyntaxKind::As),
```

Reply:
ahh sorry - the order is not relevant. I fixed the test suite in #62 to ignore the order when comparing the vectors. it should pass after you merge main into you branch. thanks!